src/main/java/lotto/domain/lotto/Lotto.java

```diff
@@ -0,0 +1,58 @@
+package lotto.domain.lotto;
+
+import java.util.List;
+import java.util.Optional;
+import lotto.constant.errorMessage.lotto.LottoExceptionStatus;
+import lotto.domain.lotto.validator.LottoValidator;
+import lotto.domain.rank.Rank;
+
+public class Lotto {
+
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        validate(numbers);
+        LottoValidator.validateNumbers(numbers);
+        this.numbers = sortNumbers(numbers);
+    }
```

Comment:
정적 팩토리 메서드를 사용해보는건 어떠신가요!?

```suggestion
    private Lotto(List<Integer> numbers) {
        validateSize(numbers);
        validateDuplicate(numbers);
        validateRange(numbers);

        this.numbers = numbers;
    }

    public static Lotto create(List<Integer> numbers) {
        return new Lotto(numbers);
    }
```

정적 팩토리 메서드는 객체 생성 로직을 클래스 내부에 숨기는 역할 즉, 캡슐화를 통해 코드를 객체 지향적으로 작성하는 데 도움을 주기 때문에 적용해 보는 것도 좋을 것 같아요!

Reply:
좋은 정보 감사합니다. 정적 팩토리 메서드에 대해서 조금 더 공부해 봐야겠네요. 😄